pkgs/applications/misc/dra/default.nix

```
@@ -0,0 +1,24 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "dra";
+  version = "2022-08-13";
```

```suggestion
  version = "unstable-2022-08-13";
```
See https://nixos.org/manual/nixpkgs/unstable/#sec-package-naming